public/javascripts/Gallery/src/filter/CityMenu.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * City Menu module.
+ * This is responsible for holding the buttons allowing users to filter citys by city type.
+ *
+ * @param uiCityMenu UI element corresponding to CityMenu.
+ * @returns {CityMenu}
+ * @constructor
+ */
+function CityMenu(uiCityMenu) {
+    let self = this;
+
+    let status = {
+        currentCityType: null
+    };
+
+    /**
+     * Initialize CityMenu.
+     */
+    function _init() {
+        if (uiCityMenu) {
+            uiCityMenu.select.bind({
+                change: handleCitySelectSwitchChangeCallback
+            })
+        }
+    }
+
+    /**
+     * Handles what happens when a city type is selected.
+     */
+    function handleCitySelectSwitchChangeCallback() {
```

Comment:
Okay these long names are bugging me (which I know you got from the code that's already there) :sweat_smile: Could you change all those function names?
* `handleSortSwitchClickCallback -> sortSelectCallback`
* `handleCitySelectSwitchChangeCallback -> citySelectCallback`
* `handleLabelSelectSwitchChangeCallback -> labelSelectCallback`
* `handleTagClickCallback -> tagClickCallback`